app/views/meditations/index.html.slim

```diff
@@ -1,99 +1,80 @@
 
 - content_for :title, @static_page.title
 
-.container
-    .breadcrumbs test
-
-section.with.half.left.blue.gradient.page-title
+section.intro
   .container
-    /h1.title = Meditation.model_name.human(count: -1)
-    h1.title Meditate Now
+    h1.title = @static_page.title
 
-section.meditation-video
+section.featured-meditations
   .container
-    .grid-row
-      - @meditations.sample(3).each do |meditation|
-        .grid-sizer
-          .meditation-item
-            h3.meditation-header
-              = meditation.name
-            .video
-              .video-preview style="background-image: url(#{meditation.image.url})"
-                img src="#{meditation.image.url}"
-                a.video.button data-mfp-src="#video-popup" href="/uploads/attachment/file/40/video.mp4"
-              h4.video-title Boost your confidence
-              p Сахаджа Йога является методом получения реального опыта самосозерцания и мировозрения.  
-
-section.own-meditation
+    - meditation = @meditations.sample
+    .meditation-item
+      h3.meditation-header
+        | Meditation of the day
+      .video
+        .video-preview style="background-image: url(#{meditation.image.url})"
+          img src="#{meditation.image.url}"
+          a.video.button href=meditation_url(meditation)
+        h4.video-title = meditation.name
+        p Сахаджа Йога является методом получения реального опыта самосозерцания и мировозрения.
+    - meditation = @meditations.sample
+    .meditation-item
+      h3.meditation-header
+        | Trending now
+      .video
+        .video-preview style="background-image: url(#{meditation.image.url})"
+          img src="#{meditation.image.url}"
+          a.video.button href=meditation_url(meditation)
+        h4.video-title = meditation.name
+        p Сахаджа Йога является методом получения реального опыта самосозерцания и мировозрения.
+    - meditation = @meditations.sample
+    .meditation-item
+      h3.meditation-header
+        | Let Fate Decide
+      .video
+        .video-preview style="background-image: url(#{meditation.image.url})"
+          img src="#{meditation.image.url}"
+          a.video.button href=meditation_url(meditation)
+        h4.video-title Generate a random meditation
+        p Сахаджа Йога является методом получения реального опыта самосозерцания и мировозрения.
```

Comment:
I basically just changed the way that this section is populated. Not really anything you need to worry about, more changes will be needed on the backend to make this section properly functional.